### ISSUE TITLE
Libretro Database Thumbs?

### ISSUE THREAD

**Issue by HerbFargus** [sselph/scraper#102]
Seems libretro has a database of images for thumbnails to be displayed with their xmb menu 

https://github.com/libretro/libretro-thumbnails

Is there a viability of including these images somehow?

**Comment by sselph** [comment]
Looks like it. I remember chatting with them and some of there work is based on mine. Notice the hash.csv and gdb scraper. Everything is keyed off the no-intro name so should line up since I can look that up based on hash. I'll just need to figure out how to integrate that option in to the dozens of options already available. If you have ideas let me know.

**Comment by HerbFargus** [comment]
Perhaps have a flag for the database (I think for the metadata they use thegamesdb perhaps?):

-use_libretro

For each system they have boxart, snaps, titles 

Don't know if it can be adapted to be included as the same flags with -gdb_img (or added as a new one called -libretro_img) with titles being added as "t" unless the title screens fall under logos.

E.g.
-libretro_img "b,s,t"

Idk just some thoughts

**Comment by sselph** [comment]
Yeah I didn't think much about multiple data sources when setting this all up so it all seems a bit kludgy.  That might work `-libretro_img`. if you don't pass it things work as normal. If you do, then it will try to get libretro images with metadata from the normal source.

**Comment by HerbFargus** [comment]
Sounds good to me

**Comment by robertybob** [comment]
Is the upshot of this that TGDB won't be using as much bandwidth as these images/thumbnails are now sourced elsewhere? Can only be a good thing I imagine :)

**Comment by lilbud** [comment]
I really hope this gets added. I downloaded the whole set of thumbs. and send them over manually. I am glad I don't have to do this for a full MAME set

**Comment by sselph** [comment]
I apologize for not updating this. I did start work on this but the way I have things written it didn't lend itself to swapping out the image source easily. I start rearranging things but didn't finish before my son woke up and haven't gotten back to this. It seems like alternate data sources is a popular request so I'll try and make some time to work on this again.

**Comment by HerbFargus** [comment]
There's no deadline, take care for the most important things in life first :)

**Comment by Kivutar** [comment]
Hello, you can get the individual images there: http://thumbnails.libretro.com/
And the packs there: http://thumbnailpacks.libretro.com/

We follow the no-intro names exactly. For systems that are not supported by no-intro, like arcade games, it's still a work in progress.

**Comment by markwkidd** [comment]
Update: in terms of arcade thumbnail support, the libreto repository now has a set of Final Burn Alpha thumbnails for 0.2.97.38/39 as well as a comprehensive MAME thumbnail set. The MAME set supports the libretro MAME cores with the most development effort: MAME 2000/0.37b5, 2003/0.78, 2010/0.139, 2014/0.159, and current 0.179.

After some cleanup last week, all thumbnail files in the repo should also be standardized to use No-Intro filename convention in terms of allowed characters for filenames. Anything that No-Intro/RetroArch doesn't allow is transformed to an underscore for thumb files.

**Comment by lilbud** [comment]
Was support for libretro-thumbnails ever added to this scraper?

**Comment by sselph** [comment]
No, I haven't done this.
